src/routes/at-sce.routes.js

```diff
@@ -12,6 +12,7 @@
 const express = require("express");
 const router = express.Router();
 const path = require("path");
+const helper = require('../helpers/auth.helper')
```

Comment:
try this 
`// Helpers
const { isAuthenticated } = require("../helpers/auth.helper");`